lib/utils.pm

```diff
@@ -3298,13 +3299,17 @@ sub render_autoinst_url {
 
     croak("Can not render autoinst url from empty url") if (!$args{url});
     if ($args{url} =~ /^(http|https)\:\/\/openqa\./im) {
+        if ($args{url} =~ /^(http|https)\:\/\/openqa\.[^\s]+\/assets\/repo\//im) {
+            record_info("Can not render autoinst url for repo assets", "openQA only syncs iso/hdd assets.Return original $args{url}", result => 'fail');
```

Comment:
So this is expected behavior to not render repo url. Suggest to remove `result => 'fail'` and keep only record_info here , which will avoid people's confusion why it is RED here.